metrics_ci_integration/test/jenkins/client/model/jenkins_query_limits_test.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:ci_integration/jenkins/client/model/jenkins_query_limits.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("JenkinsQueryLimits", () {
+    test(".empty() should create an empty range specifier", () {
+      const limits = JenkinsQueryLimits.empty();
+      const expected = '';
+      final actual = limits.toQuery();
+
+      expect(actual, equals(expected));
+    });
+
+    test(".at(n) should create a range specifier for n-th element - {n}", () {
+      final limits = JenkinsQueryLimits.at(1);
```

Comment:
We should add more tests according to the correct contract (should be positive). This should be applicable to all methods as we mostly require positive numbers here.